book/src/drivers.md

```diff
@@ -25,6 +25,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [BizFlyCloud Block Storage](https://github.com/bizflycloud/csi-bizflycloud) | `volume.csi.bizflycloud.vn` | v1.2 | A Container Storage Interface (CSI) Driver for BizFly Cloud block storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion
 [CephFS](https://github.com/ceph/ceph-csi) | `cephfs.csi.ceph.com` | v0.3, >=v1.0.0 | A Container Storage Interface (CSI) Driver for CephFS | Persistent | Read/Write Multiple Pods | Yes | Expansion, Snapshot, Cloning
 [Ceph RBD](https://github.com/ceph/ceph-csi) | `rbd.csi.ceph.com` | v0.3, >=v1.0.0 | A Container Storage Interface (CSI)  Driver for Ceph RBD | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Topology, Cloning,In-tree plugin migration
+[Cisco HyperFlex CSI](https://github.com/ciscohyperflex/csi-plugin) | `HX-CSI` | v1.2 | A Container Storage Interface (CSI)  Driver for Cisco HyperFlex | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Expansion, Cloning
```

Comment:
There seems to be 2 spaces between "(CSI)" and "Driver".  Can you remove 1?

Reply:
No problem!